### ISSUE TITLE
Doesn't work when using the jquery.hammer plugin

### ISSUE THREAD

**Issue by stebun** [josdejong/propagating-hammerjs#4]
How can I get this to work with the jquery plugin for hammer 2.0?

If I use the hammer plugin propagation doesn't work. If I don't use the hammer plugin hammer propagation does work with your propagation.js script

**Comment by stebun** [comment]
I have two elements, parent and child working with tap. 

currently Im testing the event object in my parent for event.isTrigger === undefined.

If this property isn't there I know this event is caused directly by the parent tap, if it is there, it means that the child tap event was called and I can ignore this tap on the parent.

I also need to set domEvents: true on the call to the hammer plugin "hammer({domEvents: true})" for the parent (not the child);

I'm not sure how reliable this is cross browser or how it would work with a child within a child, but for my current needs it works.

Still it would be nice if I could use the jquery plugin and use the propagation.js

**Comment by josdejong** [comment]
It may work if you first load the `propagating-hammer.js` script, then override the global `Hammer` instance (like in [this example](https://github.com/josdejong/propagating-hammerjs/blob/master/examples/global.html)), and last load `jquery.hammer.js`. Did you try that?

**Comment by stebun** [comment]
Yes this worked, but it's annoying to have to load the plugin afterwards. Can't you do anything to fix this? There must be lots of people using the widget and your propagation.js

**Comment by josdejong** [comment]
I can think of two options:
- Create a new module `jquery.propagating.hammer.js` which combines both.
- Use vanilla JavaScript instead of `jquery.hammer.js`. There isn't really much need for jQuery anymore, and JavaScript isn't that scary ;)
